dev-docs/bidders/taboola.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: bidder
+title: Taboola
+description: Prebid Taboola Bidder Adapter
+pbjs: true
+biddercode: taboola
+gdpr_supported: true
+usp_supported: true
+coppa_supported: true
+schain_supported: true
```

Comment:
Should this be removed or set to false, I am not seeing schain being passed in requests in your [initial release PR](https://github.com/prebid/Prebid.js/pull/8385/files)